actions/earned-income-budgeting.vbs

```diff
@@ -1345,11 +2361,12 @@ For ei_panel = 0 to UBOUND(EARNED_INCOME_PANELS_ARRAY, 2)       'looping through
                                     'the income needs to apply to at least one program
                                     If EARNED_INCOME_PANELS_ARRAY(apply_to_SNAP, ei_panel) = unchecked AND EARNED_INCOME_PANELS_ARRAY(apply_to_CASH, ei_panel) = unchecked AND EARNED_INCOME_PANELS_ARRAY(apply_to_HC, ei_panel) = unchecked AND EARNED_INCOME_PANELS_ARRAY(apply_to_GRH, ei_panel) = unchecked Then sm_err_msg = sm_err_msg & vbNewLine & "* No programs have been selected that this icnome applies to. Chose at least one program that this income is budgeted for."
                                     EARNED_INCOME_PANELS_ARRAY(verif_date, ei_panel) = trim(EARNED_INCOME_PANELS_ARRAY(verif_date, ei_panel))
-                                    If EARNED_INCOME_PANELS_ARRAY(verif_date, ei_panel) = "" Then sm_err_msg = sm_err_msg & "* Enter the date the pay information was received in the agency."
+                                    If EARNED_INCOME_PANELS_ARRAY(verif_date, ei_panel) = "" Then sm_err_msg = sm_err_msg & vbNewLine & "* Enter the date the pay information was received in the agency."
+									If split_pay_btn_pressed = True and Left(sm_err_msg, 4) <> "LOOP" Then sm_err_msg = sm_err_msg & vbNewLine & "* Button was pressed to enter Split Pay information, but suffiencent detail was not entered into the paycheck to enter split pay information."
```

Comment:
Line 2365: Spelling correction "but suffiencent detail" should be "sufficient"

Reply:
Thank you!